apps/core/components/fromTo/FromToWrapper.js

```diff
@@ -0,0 +1,23 @@
+// @flow
```

Comment:
Do we really need a component for this? Maybe the style could be added to FromTo component instead?

Reply:
The thing is that OneWay and multicity can Reuse the TripInfo as is. But return is slightly different, so it needs to reuse the internals of the TripInfo component. 

So instead of copy/pasting this style and having it duplicated, I extracted it.

Reply:
Okay, why not.